macros/amla.cfg

```diff
@@ -147,11 +158,15 @@
             increase=1
         [/effect]
         [effect]
-            heal_full=yes
+            apply_to=hitpoints
+            increase_total=-10%
+            [filter]
+                type="TLU_Mehir_Guard,TLU_Mehir_Commander"
+            [/filter]
         [/effect]
         [effect]
-            apply_to=movement
-            increase=50
+            apply_to=hitpoints
+            heal_full=yes
```

Comment:
Having an HP penalty is inconsistent with healing the unit.

Reply:
Unless player is informed about it, amla should include healing

Reply:
Then again, AMLAs arguably shouldn't include penalties. Players *are* informed of HP penalty, however, so it wouldn't be surprising that they don't heal. That's a bit of a conflicting design issue with BfW.

Still, there's something that is definitely a bug in the live version. Since the AMLA can be taken under penalty-less conditions, it should 100% perform healing there.